android_git.xml

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"

```

Comment:
Instead of creating the same layouts use one layout and set images/text programmically.